crates/biome_cli/src/cli_options.rs

```diff
@@ -84,6 +84,10 @@ pub struct CliOptions {
     )]
     /// The level of diagnostics to show. In order, from the lowest to the most important: info, warn, error. Passing `--diagnostic-level=error` will cause Biome to print only diagnostics that contain only errors.
     pub diagnostic_level: Severity,
+
+    #[bpaf(long("use-editorconfig"), argument("true|false"), optional)]
+    /// Use any `.editorconfig` files to configure the formatter. Configuration in `biome.json` will override `.editorconfig` configuration.
+    pub use_editorconfig: Option<bool>,
```

Comment:
I think users would also want to control the `.editoconfig` from the configuration file `biome.json`. If you think about it, this isn't related to the CLI (like colours, the verbosity of diagnostics, etc.) but it's something related to the formatter.

Maybe this option should be placed inside `formatter.useEditorconfig`, next to `formatter.enabled`.